modules/sriov-operator-hosted-control-planes.adoc

```diff
@@ -14,7 +15,7 @@ After you configure and deploy your hosting service cluster, you can create a su
 
 .Prerequisites
 
-You have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/clusters/cluster_mce_overview#hosted-control-planes-configure[configured and deployed] the hosted cluster.
+You have link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/configure_hosted_aws#hosting-service-cluster-configure-aws[configured and deployed] the hosted cluster.
```

Comment:
I noticed in other files that a past tense tone is used. I'm more so suggesting to write out the exact section title, rather than hyperlinking the words "configure and deployed"
```suggestion
You must configure and deploy the hosted cluster on AWS. For more details see link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.8/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring the hosting cluster on AWS (Technology Preview)]
```